swan-lake/learn-the-platform/ballerina-persist/bal-persist-overview.md

```diff
@@ -5,11 +5,24 @@ description: The section gives an overview of the `bal persist` feature.
 keywords: ballerina, programming language, ballerina packages, persist, data model, cli tool, client api
 permalink: /learn/ballerina-persist/persist-overview/
 active: persist_overview
-intro: The `bal persist` feature allows you to store data in different data stores and retrieve them when needed. A data store can be a database, an in-memory cache, or a file system. The `bal persist` feature currently supports in-memory tables, MySQL databases, and Google Sheets as data stores. The important point is you can use the same syntax to access data in all these data stores. Therefore, you don't need to learn different syntaxes to access data in different data stores.
+intro: The `bal persist` feature allows you to store data in different data stores and retrieve them when needed. A data store can be a database, an in-memory cache, or a file system. The `bal persist` feature currently supports in-memory tables, MySQL and MSSQL databases, and Google Sheets as data stores. The important point is you can use the same syntax to access data in all these data stores. Therefore, you don't need to learn different syntaxes to access data in different data stores.
 redirect_from:
 - /learn/ballerina-persist/persist-overview/
 ---
-This feature has three main components: Persist Model, Persist CLI, and Persist Client API. The Persist Model is used to define the data model. The Persist CLI is used to generate the client API for the data model. The Persist Client API is used to access the data in the data store.
+This feature has three main components: Data Model, CLI tool, and Type-safe client API. 
+* The data model definition is used to define the data model. 
+* The CLI tool is used to generate the client API for the data model. 
+* The type-safe client API is used to access the data in the data store.
+
+The following diagram illustrates how the `bal persist` feature works.
+
+![bal persist diagram](/learn/images/bal-persist-diagram.png "bal persist diagram")
```

Comment:
The image link seems to be broken?

```suggestion
![bal persist diagram](/public/learn/images/bal-persist-diagram.png "bal persist diagram")
```

Reply:
@praneesha do we need `public` part on the page link

Reply:
No, we don't.